### DISCUSSION TITLE
Compatibility with Windows Sandbox

### DISCUSSION THREAD

**Discussion by djoate** [mullvad/mullvadvpn-app#9067] [Feature requests & Ideas]
Currently, I cannot get Mullvad to work with [Windows Sandbox](https://techcommunity.microsoft.com/t5/windows-kernel-internals/windows-sandbox/ba-p/301849). When running the Mullvad VPN app as the host, the sandbox never has internet connection when the Mullvad VPN app is active. I can get internet access inside of the sandbox if I disconnect from Mullvad.

**Comment by faern** [comment]
I'm just guessing now. But I suppose the sandbox uses some kind of virtual network to talk to the host? If so, our firewall rules might be blocking all such traffic. Try enabling "Local network sharing" in the Mullvad VPN app and see if it helps. If not, they are either using some non-private address space to do this virtual networking, or it's something completely different that is the issue.

**Comment by djoate** [comment]
Local network sharing enabled still results in the sandbox having no internet connection
